jobs/JGDAS_ENKF_FCST

```diff
@@ -52,7 +52,6 @@ status=$?
 export CDATE=${CDATE:-${PDY}${cyc}}
 export CDUMP=${CDUMP:-${RUN:-"gdas"}}
 export COMPONENT="atmos"
-export rCDUMP="gdas"
```

Comment:
This hard coded line is for gfsefmn jobs in early enkf. This will cause jobs can not find the data in previous cycle enkf/gdas

Reply:
You are right @XianwuXue-NOAA 
My apologies for missing this in the review. 
We can open a hotfix asap.

Reply:
No problem. the gdas[gfs]anal[eobs] jobs also failed, after reverse the changes in exglobal_atmos_analysis.sh, I can run it successfully. I tried to change crtm_ver from 2.3.0 to 2.4.0, however, it does not pass the value. I think I modified the wrong file (ecf/versions/gfs.ver).

Reply:
The initialize of the rCDUMP is for rCDUMP used in forecast_predet.sh that is default to CDUMP.
A GW meeting has decision made to revert this change.